paasta_tools/cli/cmds/spark_run.py

```diff
@@ -538,6 +577,12 @@ def get_spark_app_name(original_docker_cmd: Union[Any, str, List[str]]) -> str:
         spark_app_name = "paasta_spark_run"
 
     spark_app_name += f"_{get_username()}"
```

Comment:
I think this was missed in the past review, what happens if the user is batch or service or nobody?

Reply:
I think this is the way historically been done, see https://github.com/Yelp/paasta/blame/8b87e0b13816c09b3d063b6d3271e6c7627fd264/paasta_tools/cli/cmds/spark_run.py#L540, guess if it's service name or nobody that would be attached to the job as well